utils/dockerfile_assets/github_dl.py

```
@@ -34,6 +34,30 @@ def validate_binary(binary, checksum, raw_algorithm="sha256") -> bool:
     return True
 
 
+def validate_token(token) -> bool:
+    headers = {
+        "Authorization": f"Bearer {token}",
+        "Accept": "application/vnd.github+json",
+        "X-GitHub-Api-Version": "2022-11-28",
+    }
+
+    response = requests.get("https://api.github.com/rate_limit", headers=headers)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)